packages/lodestar/src/metrics/metrics/lodestar.ts

```diff
@@ -68,10 +68,6 @@ export function createLodestarMetrics(
       help: "Total number of goodbye sent, labeled by reason",
       labelNames: ["reason"],
     }),
-    peersTotalUniqueConnected: register.gauge({
-      name: "lodestar_peers_total_unique_connected",
```

Comment:
we need to remove this from `lodestar.json` too

Reply:
We can do that latter, many charts need adjustments in that row